admin_guide/managing_networking.adoc

```diff
@@ -1417,6 +1418,21 @@ new project is created. To do this:
 . Create a custom project template and configure the master to use it, as
 described in
 xref:../admin_guide/managing_projects.adoc#modifying-the-template-for-new-projects[Modifying the Template for New Projects].
+
+. Label the `default` project with the `default` label:
++
+[IMPORTANT]
+====
+Skip this step if you have labeled the default project with the `default` label
```

Comment:
If you labeled the default project with the `default` label in a previous procedure, then skip this step.